data/en/sizeof.json

```diff
@@ -4,14 +4,14 @@
 	"syntax": "sizeOf(obj [, complex])",
 	"returns": "any",
 	"related": [],
-	"description": "this function is experimentell, to not use in productive enviroment. this function returns the size in bytes of a given object",
+	"description": "Returns the size in bytes of a given object",
 	"params": [
 		{"name": "obj", "description": "", "required": true, "default": "", "type": "any", "values": []},
 		{"name": "complex", "description": "", "required": false, "default": "", "type": "boolean", "values": []}
 	],
 	"engines": {
-		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/sizeof.html"}
+		"lucee": {"minimum_version": "", "notes": "", "discouraged":"This function is in an early state. Do not use in production!","docs": "http://docs.lucee.org/reference/functions/sizeof.html"}
```

Comment:
The `discouraged` key goes on the top level object, not the engine object since it is an opinion covering the feature. Example: https://github.com/foundeo/cfdocs/blob/b6d6968e86f6e105da8430091e39fc798266aa38/data/en/cfpod.json#L8